tools/accuracy_checker/configs/googlenet-v4-tf.yml

```diff
@@ -0,0 +1,45 @@
+models:
+  - name: googlenet-v4-tf
+    launchers:
+      - framework: tf
+        model: public/inception_v4.frozen.pb
+        adapter: classification
+
+    dataset: 
+      - name: imagenet_1001_classes
+        preprocessing:
+          - type: bgr_to_rgb
+          - type: normalization
+            mean: [127.5,127.5,127.5]
+            std: 127.5
+          - type: resize
+            size: 300
```

Comment:
are you sure about 300? crop only one pixel looks strange :)

Reply:
Doe it should not be a standart inception preprocessing (crop central fraction 0.875 + resize to input shape)?
https://github.com/tensorflow/models/blob/master/research/inception/inception/image_processing.py#L279